### PR TITLE
fix: lowercase token addresses directories

### PR DESCRIPTION
**Pull request by shoom3301**
1. Lowercased existing directories ([commit](https://github.com/cowprotocol/token-lists/pull/287/commits/90d5f646393fe34072baf92b54a5d983f98ebbc9))
2. Lowercased `logoURI` in `info.json` files ([commit](https://github.com/cowprotocol/token-lists/pull/287/commits/c938d4ccb0c40b9fa4eec752a8665c28c240b3ab))
3. Lowercased `logoURI` in `CowSwap.json` ([commit](https://github.com/cowprotocol/token-lists/pull/287/commits/f6342f8a390744837b758b55617926094e5144ae))
4. Added token address lowercasing in `executeAction.yml` ([commit](https://github.com/cowprotocol/token-lists/pull/287/commits/172cf26f0dabe445310456ac83a493025d9f1210))

### CODE REVIEW

File: .github/workflows/executeAction.yml [hunk 1]
```diff
@@ -28,8 +28,13 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      # https://stackoverflow.com/questions/62627931/github-actions-expression-functions-string-manipulation
+      - name: Set address lowercase
+        id: setAddressLowerCase
+        run: INPUT=${{ fromJSON(inputs.issueInfo).address }} echo "::set-output name=addressLowerCase::${INPUT,,}"
+
```
Comment (alfetopito) [inline .github/workflows/executeAction.yml:35]: Awesome!

File: src/public/CowSwap.json [hunk 2]
```diff
@@ -891,63 +891,63 @@
...
+      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48/logo.png"
     },
     {
       "address": "0xE2e73A1c69ecF83F464EFCE6A5be353a37cA09b2",
       "chainId": 100,
       "name": "ChainLink Token on Gnosis Chain",
       "symbol": "LINK",
       "decimals": 18,
-      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0x514910771AF9Ca656af840dff83E8264EcF986CA/logo.png"
+      "logoURI": "https://raw.githubusercontent.com/centfinance/assets/master/blockchains/ethereum/assets/0x514910771af9ca656af840dff83e8264ecf986ca/logo.png"
```
Comment (alfetopito) [inline src/public/CowSwap.json:942]: Not all of them can be auto-lower cased, unfortunately.
This one for example results in a 404.

The best approach would be to lower case the `logoURI` ONLY if it's pointing to this repo.

File: src/public/CowSwapGoerli.json [hunk 3]
```diff
@@ -15,55 +20,55 @@
...
+      "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x6810e776880c02933d47db1b9fc05908e5386b96/logo.png"
     },
     {
       "name": "CoW Protocol Token",
       "chainId": 5,
       "symbol": "COW",
       "decimals": 18,
       "address": "0x91056D4A53E1faa1A84306D4deAEc71085394bC8",
-      "logoURI": "https://gnosis.mypinata.cloud/ipfs/Qme9B6jRpGtZsRFcPjHvA5T4ugFuL4c3SzWfxyMPa59AMo"
+      "logoURI": "https://gnosis.mypinata.cloud/ipfs/qme9b6jrpgtzsrfcpjhva5t4ugful4c3szwfxympa59amo"
```
Comment (alfetopito) [inline src/public/CowSwapGoerli.json:31]: Another case that should not be lower cased 😅

**Review by alfetopito — CHANGES_REQUESTED** [review]
Thanks for this, I should have done this in the first place 🙈 

BTW, not all token image urls can be lower cased. See the comments.

Also, this should be updated too https://github.com/cowprotocol/token-lists/blob/d4de444f3a27bb58caac805cceeed2cc99cdb9a0/src/scripts/workflow_helper.py#L69

File: src/public/CowSwap.json (hunk 2 above)
Comment (shoom3301) [reply src/public/CowSwap.json:942]: Damn, really, thanks!

Comment (shoom3301) [reply src/public/CowSwap.json]: FIxed

File: src/public/CowSwapGoerli.json (hunk 3 above)
Comment (shoom3301) [reply src/public/CowSwapGoerli.json]: Thanks!
Fixed them all.
I actually did all fixes using scripts, so I reverted my changes, fixed script and ran the script again

**Comment by shoom3301** [comment]
> Thanks for this, I should have done this in the first place 🙈
> 
> BTW, not all token image urls can be lower cased. See the comments.
> 
> Also, this should be updated too
> 
> https://github.com/cowprotocol/token-lists/blob/d4de444f3a27bb58caac805cceeed2cc99cdb9a0/src/scripts/workflow_helper.py#L69

@alfetopito thanks! fixed it as well

**Comment by shoom3301** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by alfetopito — APPROVED** [review]

File: .github/workflows/executeAction.yml (hunk 1 above)
Comment (anxolin) [reply .github/workflows/executeAction.yml:35]: Great you added also the stackoverflow ref!

**Pull request merged**